### DISCUSSION TITLE
Project HÆLTH Blog #1

### DISCUSSION THREAD

**Discussion by ldcdorn** [ldcdorn/haelth#12] [General]
# Project HÆLTH 🌱
Introducing Project HÆLTH, an open source mobile app which helps you with your personal nutrition, exercise, and mindfulness goals.

## The Team 🤝
| Name                    | Role(s)                                  |
| ------------------------| ---------------------------------------|
| Leon Dorn            |Scrum master, Developer   |
| Sid Müller            |Product Owner, Developer |
| Omar Moujahed |Developer                              |
| Jan Tibke              |Developer                              |

## Vision 🌠
HÆLTH creates tailored, science-based fitness and meditation routines for you, based on your priorities and personal profile. HÆLTH supports you with tracking and improving your general health. With HÆLTH, your data IS and REMAINS all yours, locally on your device.
HAELTH gives you full control by providing multiple options to export your data. This means you can do further analysis on you workout data in a different program and migrate your data to another app at any time, just as you see fit.

## Technology 🤖
* Project HÆLTH is developed for Android with Kotlin.
* By leveraging (Kotlin Multiplatform)[https://kotlinlang.org/docs/multiplatform.html], we expect to port it to other Platforms in the future (no promises).

## Scrum
Have a look at our (projects  page)[https://github.com/ldcdorn/haelth/projects] to see our scrum boards.
For example: here are the links to our (Product Backlog)[https://github.com/users/ldcdorn/projects/4] 📒 and (current sprint)[https://github.com/users/ldcdorn/projects/7] 🏃

| Scrum Event          | Duration    |
| -------------------- | ----------- |
| Sprint               | 2 weeks     |
| Sprint Planning      | 30 minutes* |
| Daily Scrum          | 10 minutes* |
| Sprint Review        | 15 minutes* |
| Sprint Retrospective | 15 minutes* |

**Comment by Ninetilt** [comment]
Uns gefällt besonders gut an Projekt HÆLTH, dass es sich nicht nur auf körperliche Bewegung fokussiert, sondern auch gesunde Ernährung und andere wichtige Aspekte der Gesundheit, wie Achtsamkeit, mit einbezieht. Diese ganzheitliche Herangehensweise macht die App besonders wertvoll. Auch die lokale Speicherung der Daten und die Möglichkeit, sie flexibel zu exportieren, finden wir sehr gelungen.

Viele Grüße
Team TDC (JS)

**Comment by JustMischka** [comment]
Project HÆLTH hat einen spannenden Ansatz, um das Thema Gesundheit anzugehen, besonders mit der Kombination aus Fitness, Meditation und persönlicher Datenkontrolle. Dass eure App dabei komplett Open Source ist und den Datenschutz ernst nimmt, finde ich richtig stark. Mir gefällt auch, dass ihr die Möglichkeit bietet, Daten zu exportieren und sie in anderen Programmen weiterzuverwenden – das gibt den Nutzern wirklich Flexibilität. Gute Arbeit!

Viele Grüße
Team Jump and Run (MG)
